test/unit/dom/dom.js

```diff
@@ -20,32 +20,121 @@ describe('skatejs-named-slots dom', function () {
 
     // Now it has something to assign nodes to.
     root.appendChild(slot);
+
+    div1 = document.createElement('div');
+    div2 = document.createElement('div');
+    div3 = document.createElement('div');
+    div4 = document.createElement('div');
+    div5 = document.createElement('div');
+
+    div1.appendChild(div2);
+    div3.appendChild(div4);
```

Comment:
Might be nicer if this line was moved down one – then it'd read as a pre-order traversal of the tree :P